TK-Logo-Combined.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Correct, that was by mistake, it is the logo from tankerkoenig.de. I'l remove it.